src/redux/greetings/greetingSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchGreeting = createAsyncThunk('greeting/fetchGreeting', async () => {
+  const response = await axios.get('http://127.0.0.1:3000/api/random_greeting');
```

Comment:
- [ ] Great job so far @adeola003 There are a few changes I will suggest you implement and you'll be on your way to the next milestone :wink:  
- [ ] I noticed that your `Frontend` followed all the project requirements, however as a result of the error from your `Backend`, the two apps are not able to work together as intended. So kindly fix this as well so the error below can be solved and when your app is navigated to the `greeting route`, a random greeting will be shown on the screen instead of this error message below :wink: 
- Error Message
![Screenshot from 2023-08-17 18-19-09](https://github.com/adeola003/greeting_ui/assets/102699958/890ace3f-6a35-4611-8877-8d54ddbb52c3)